final/src/main/java/ua/epam/spring/hometask/converter/Converter.java

```diff
@@ -0,0 +1,38 @@
+package ua.epam.spring.hometask.converter;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public interface Converter<E, D> {
+    E createFromDTO(D dto);
+
+    D createFromEntity(E entity);
+
+    E updateEntity(E entity, D dto);
```

Comment:
As far as I see it, this method is used for internal purposes so there is no reason to make it public. Moreover, there in no need in this method, see comments below.